tools/ci/check_grep.sh

```diff
@@ -69,6 +69,16 @@ if grep -Pzo '"\w+" = \(\n[^)]*?/obj/structure/chair/(?<type>[/\w]*),\n[^)]*?/ob
     echo "ERROR: found multiple identical chairs on the same tile, please remove them."
     st=1
 fi;
+if grep -Pzo '"\w+" = \(\n[^)]*?/obj/machinery/door/airlock/(?<type>[/\w]*),\n[^)]*?/obj/machinery/door/airlock/\g{type},\n[^)]*?/area/.+\)' _maps/**/*.dmm;	then
```

Comment:
```suggestion
if grep -Pzo '"\w+" = \(\n[^)]*?/obj/machinery/door/airlock[/\w]*?,\n[^)]*?/obj/machinery/door/airlock[/\w]*?,\n[^)]*?/area/.+\)' _maps/**/*.dmm;	then
```
Haven't touched Perlre in forever so beware but it should help detect more valid cases where different subtypes are still on the same tile. Potentially turn both grep checks into a single one for /obj/machinery/door/ as I can't think of an instance where we want both windoors and airlocks on the same tile, or any other subtypes of it.